### PR TITLE
Active Tab fix 

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #11648
### Summary of Changes

Do not keep adding tabs, active should be only one
### Testing Instructions

Apply patch and navigate to administrator area on views with tabs and reload pages after selecting different tab. The active tab should be always the last selected.
### Documentation Changes Required

None

### CODE REVIEW

File: media/system/js/tabs-state.js [hunk 1]
```diff
@@ -37,6 +36,7 @@ jQuery(function($) {
         var $tabs = $('a[data-toggle="tab"]');
 
         $tabs.on('click', function(e) {
+            console.log($(e.target).attr('href'));
```
Comment (ggppdk) [inline media/system/js/tabs-state.js]: You forgot testing code here ?

Comment (zero-24) [inline media/system/js/tabs-state.js]: Is this for the final version or just debug code?

Comment (dgt41) [inline media/system/js/tabs-state.js]: Fixed it, thanks

**Comment by BurtNL** [comment]
I applied this patch on 3.6.3-dev, but it doesn't solve the issue as reported in #11648.
The active selected tab is still random, sometimes the first one, sometimes the last one and sometimes even with no active tab visible.

screen shot 2016-08-18 at 10 01 04

After applying the patch I cleared all Joomla-caches and the browser cache, still the same result as described above.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11658">issues.joomla.org/joomla-cms/11658</a>.</sub>

**Comment by dgt41** [comment]
@BurtNL you have to clear your local storage or do some tab selection

**Comment by BurtNL** [comment]
@dgt41 I re-installed the latest 3.6.3-dev, applied the patch, cleared the browser cache and Joomla caches, closed the browser (FF48).
Still the same problem, the active tab is randomly selected and sometimes even none.

So I went to Contacts and selected the tab Permissions and then choose another subject, the tab Permissions is then always active.
Then I went to Articles and selected the tab Articles.
Then I went to Banners and no tab was selected at all.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11658">issues.joomla.org/joomla-cms/11658</a>.</sub>

**Comment by dgt41** [comment]
I am gonna need some data to understand why this is failing for you, so can you open web inspector and select the cog icon (on the right) and then enable storage?
Then selecting storage you should have something like:
screen shot 2016-08-18 at 18 59 10

Can you verify that when you click on another tab the values change and what are the new values?

**Comment by BurtNL** [comment]
After clearing browser and Joomla caches I performed the following test.
I followed your instructions and got the Global storage visible.

Started Joomla, go to Global Configuration.
The panel below shows: Key: active-tabs; Value: ["#page-site"] and Key: jsidebar; Value: false.
On selecting another tab the Key and Value changes; i.e tab System says: Key: active-tabs; Value: ["#page-system"].
So selecting other tabs changes the key and value.

Then I selected the component Articles. Nothing shows up in the panel below, besides: Key: jsidebar; Value: false
Then I selected tabs within Articles; the key and value changes on every click.

Is this the info you are looking for?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11658">issues.joomla.org/joomla-cms/11658</a>.</sub>

**Comment by dgt41** [comment]
@BurtNL That's totally strange, I mean if you have values like `["#page-site"]` then the script should work fine. Before this patch the problem was that instead of one value you could end up with an array. I have to somehow replicate your problem in order to test/debug and so far I can't...

**Comment by BurtNL** [comment]
@dgt41 This is what I see happening. When I choose a component (on left hand side) and select the tab Permissions than that selection becomes 'sticky'. Choosing another component than also has the tab Permissions selected. As long as tab Permissions is available in the component it will select it.

When I choose component Articles and select the tab Articles and than go to another component (which doesn't have a tab called Articles) than things start to go wrong. Since tab articles does not exist in i.e. Contacts it selects tab Permissions or nothing.

I guess this was already clear to you, but maybe worth mentioning.
If I need to do another test later on, let me know, happy to test it.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11658">issues.joomla.org/joomla-cms/11658</a>.</sub>

**Comment by dgt41** [comment]
@BurtNL the solution here is somehow more complicated than I initially thought. I am gonna close this and try another approach, but please do reopen the issue

**Pull request closed without merging**